docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,11 @@
+Authentication
+===
+
+1. frontend: Sends user to the lock interface that's provided by auth0 when required.
+1. frontend: Lock interface returns a json web token.
+1. frontend: When making a request to the project service it sends the web token in the authorization header.
```

Comment:
I would think on the client side, since the same token should be used for all the microservices that the front end communicates with. That was my understanding.